core/src/SYCL/Kokkos_SYCL_ParallelFor_MDRange.hpp

```diff
@@ -152,7 +153,20 @@ class Kokkos::Impl::ParallelFor<FunctorType, Kokkos::MDRangePolicy<Traits...>,
             {n_global_x, n_global_y, n_global_z},
             {global_x, global_y, global_z}, {local_x, local_y, local_z})
             .exec_range();
-      });
+      };
+#ifdef SYCL_EXT_ONEAPI_KERNEL_PROPERTIES
```

Comment:
You might want to include this in the information printed by `SYCL::print_configuration()`

Reply:
Sure.